### PR TITLE
etcdctlv3: test case: make-mirror modify dest prefix

### PR DESCRIPTION
**Pull request by sinsharat**
e2e test case added for make-mirror modify dest prefix

### CODE REVIEW

**Comment by sinsharat** [comment]
@gyuho have remove the blank line.
Thanks!

**Comment by sinsharat** [comment]
@xiang90 @heyitsanthony request you to provide feedback.
Thanks!

**Comment by xiang90** [comment]
I think we'd better to use two separate etcd clusters to test out mirroring instead of using one.

other than that, lgtm.

**Comment by sinsharat** [comment]
@xiang90 Thanks for the feedback. Sorry if i am wrong, but i think we are using two separate clusters only, each of size 1.

File: e2e/ctl_v3_make_mirror_test.go [hunk 1]
```diff
@@ -70,3 +72,54 @@ func makeMirrorTest(cx ctlCtx) {
 		cx.t.Fatal(err)
 	}
 }
+
+func makeMirrorModifyPrefixTest(cx ctlCtx) {
+	// set up another cluster to mirror with
+	cfg := configAutoTLS
```
Comment (xiang90) [inline e2e/ctl_v3_make_mirror_test.go]: mirrorCfg?

File: e2e/ctl_v3_make_mirror_test.go [hunk 2]
```diff
@@ -70,3 +72,54 @@ func makeMirrorTest(cx ctlCtx) {
...
+	cfg.clusterSize = 1
+	cfg.basePort = 10000
+	cx2 := ctlCtx{
```
Comment (xiang90) [inline e2e/ctl_v3_make_mirror_test.go]: mirrorCtx?

File: e2e/ctl_v3_make_mirror_test.go [hunk 3]
```diff
@@ -70,3 +72,54 @@ func makeMirrorTest(cx ctlCtx) {
...
+		t:           cx.t,
+		cfg:         cfg,
+		dialTimeout: 7 * time.Second,
+	}
+
+	epc, err := newEtcdProcessCluster(&cx2.cfg)
```
Comment (xiang90) [inline e2e/ctl_v3_make_mirror_test.go]: mirrorepc?

**Comment by xiang90** [comment]
@sinsharat You are right. LGTM!

**Comment by sinsharat** [comment]
@gyuho Can this be merged now?

**Comment by gyuho** [comment]
@sinsharat Could you address @xiang90's feedback? And then we can merge.

Thanks a lot!

**Comment by sinsharat** [comment]
@gyuho sorry i didn't recognise that i need to modify the names. Actually xiang mention at the end LGTM so i got confused. Will surely modify the changes.
Thanks!

File: e2e/ctl_v3_make_mirror_test.go (hunk 1 above)
Comment (sinsharat) [inline e2e/ctl_v3_make_mirror_test.go]: done

File: e2e/ctl_v3_make_mirror_test.go (hunk 2 above)
Comment (sinsharat) [inline e2e/ctl_v3_make_mirror_test.go]: done

File: e2e/ctl_v3_make_mirror_test.go (hunk 3 above)
Comment (sinsharat) [inline e2e/ctl_v3_make_mirror_test.go]: done

**Comment by sinsharat** [comment]
@xiang90 @gyuho I have done the changes as per the suggestion.
Thanks!

**Pull request merged**